packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -484,6 +482,9 @@ export class DatabaseCluster extends DatabaseClusterNew {
   private readonly singleUserRotationApplication: secretsmanager.SecretRotationApplication;
   private readonly multiUserRotationApplication: secretsmanager.SecretRotationApplication;
 
+  public readonly instanceIdentifiers: string[];
+  public readonly instanceEndpoints: Endpoint[];
```

Comment:
> They are actually moved to the implementing classes, so they no longer exist on the `DatabaseClusterNew` class. (diff looks almost as if they were just moved a bit down inside same class, but there's actually 200 rows in between)
> 
> The reason for moving them was that I could not figure out a way to keep them readonly on the base class if we needed to set them from the implementing classes.

Ah, right. I missed that. Sounds good!

Can you please move these to the block of the `public` fields above? (The one that ends with `  public readonly connections: ec2.Connections;`).

Reply:
Oops, forgot to address that part of the comment. Now it should be ok :)